conans/test/unittests/client/build/meson_test.py

```diff
@@ -71,60 +71,66 @@ def folders_test(self):
             'includedir': 'include',
             'cpp_std': 'none'
         }
-
+        build_dir = os.path.join(self.tempdir, "build")
```

Comment:
I'd expect most of the existing tests to stay unchanged, as they aren't using cross-building, and thus cross-file as well. and adding few tests specific for the cross-building to test newly added features.